### PR TITLE
Move publish_queue, vocabularies and some formatters to core.

### PR DESCRIPTION
**Pull request by amagdas**
Refactor archive_publish to publish.content module.

### CODE REVIEW

File: server/settings.py [hunk 1]
```diff
@@ -162,6 +162,7 @@ def env(variable, fallback_value=None):
     'superdesk.upload',
     'superdesk.notification',
     'superdesk.activity',
+    'superdesk.vocabularies',
```
Comment (sivakuna-aap) [inline server/settings.py]: Any specific reason for registering `superdesk.vocabularies` in both `superdesk` and `superdesk-core`? Ideally it should be registered in `superdesk-core` only.

**Comment by plamut** [comment]
In general it looks good to me, although it's a bit difficult to keep track of everything, might have overlooked something...

:+1:

File: server/settings.py (hunk 1 above)
Comment (amagdas) [inline server/settings.py]: Nop, it's not like that. The `settings.py` file from sd-core is not shared at all, that is used only for setting up the app for testing purposes.
Each app has to define complete settings as we don't want to impose on apps what modules to use.

Comment (sivakuna-aap) [inline server/settings.py]: Gotcha

**Comment by sivakuna-aap** [comment]
@amagdas whats the timeline to merge this PR? [PR#1222](https://github.com/superdesk/superdesk/pull/1222) is blocked by this.

**Comment by amagdas** [comment]
@sivakuna-aap the timeline is as soon as I get the build passing :)

**Pull request merged**